src/test/components/__snapshots__/MarkerTooltipContents.test.js.snap

```diff
@@ -157,13 +160,15 @@ exports[`MarkerTooltipContents renders tooltips for various markers: DOMEvent-10
       <div
         class="tooltipLabel"
       >
-        Thread:
+        Thread
```

Comment:
That's a weird spacing difference in the snapshot, but they still appear right next to each other in the rendered component, so it looks fine.

Reply:
yeah, I believe this is because it's rendered as 2 text nodes (or 2 virtual dom text nodes), while before we were doing the concatenation in JS so we had 1 text node only.